canonical/grafana-k8s-operator
# Make the `All` option _actually_ pick all options

## Issue
The "All" as empty string option isn't ok unless fuzzy searching is used, since `juju_application=""` won't match anything.

## Solution
Update the matchers (and injection) to use regexp searches. This is annoying in regexp searches due to the fact that we can't pass a template string to `cos-tool` (right now)


## Testing Instructions
Build a test Grafana with this library. Relate it to something (let's say Kafka). Make sure they are filled in.

## Release Notes
Make the `All` option _actually_ pick all options
